build.gradle

```diff
@@ -12,6 +12,7 @@ repositories {
 dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    //implementation 'com.AlizaBuchen:apikeys:0.2'
```

Comment:
Why is your name here?

Reply:
I was following what was on your website. by dependencies you wrote 
implementation 'com.andrewoid:apikeys:0.2'

Reply:
You are still not following what I wrote. 

I never asked you to replace it with your name. 

It's a library that I wrote that you are using.